snap/info.go

```diff
@@ -800,6 +802,7 @@ type HookInfo struct {
 
 	Environment  strutil.OrderedMap
 	CommandChain []string
+	Explicit     bool
```

Comment:
As with plugs and slots I would make this a private attribute unless there are valid reasons to make it public.